src/arm/bbai-bone-buses.dtsi

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include <dt-bindings/board/am572x-bbai-pins.h>
+
+&dra7_pmx_core {
+	bone_uart_1_pins: pinmux_bone_uart_1_pins {
+		pinctrl-single,pins = <
+			P9_24( PIN_OUTPUT_PULLUP | MUX_MODE3)	/* gpio6_15.uart10_txd */
+			P9_26A( PIN_INPUT_PULLUP | MUX_MODE3)	/* gpio6_14.uart10_rxd */
+			/* unused pins */
+			P9_26B( PIN_OUTPUT | MUX_MODE15)
+		>;
+	};
+
+	bone_uart_2_pins: pinmux_bone_uart_2_pins {
+		pinctrl-single,pins = <
+			P9_21B( PIN_OUTPUT_PULLUP | MUX_MODE1)	/* spi2_d1.uart3_txd */
+			P9_22B( PIN_INPUT_PULLUP  | MUX_MODE1)	/* spi2_sclk.uart3_rxd */
+			/* unused pins */
+			P9_21A( PIN_OUTPUT | MUX_MODE15)
+			P9_22A( PIN_OUTPUT | MUX_MODE15)
+		>;
+	};
+
+	bone_uart_4_pins: pinmux_bone_uart_4_pins {
+        pinctrl-single,pins = <
+            P9_13( PIN_OUTPUT_PULLUP | MUX_MODE4)	/* mcasp3_axr1.uart5_txd */
+            P9_11A( PIN_INPUT_PULLUP | MUX_MODE4)	/* mcasp3_axr0.uart5_rxd */
+            /* unused pins */
+			P9_11B( PIN_OUTPUT | MUX_MODE15)
+        >;
+    };
+
+	bone_uart_5_pins: pinmux_bone_uart_5_pins {
+		pinctrl-single,pins = <
+			P8_37B( PIN_OUTPUT_PULLUP | MUX_MODE3)	/* uart8_txd */
+			P8_38B( PIN_INPUT_PULLUP | MUX_MODE3)	/* uart8_rxd */
+			/* unused pins */
+			P8_37A( PIN_OUTPUT | MUX_MODE15)
+			P8_38A( PIN_OUTPUT | MUX_MODE15)			
+		>;
+	};
+};
+
+bone_uart_1: &uart10 {
+	status = "disabled";
+    pinctrl-names = "default";
+    pinctrl-0 = <&bone_uart_1_pins>;
+    symlink = "bone/uart/1";
+};
+
+bone_uart_2: &uart3 {
+	status = "disabled";
+	pinctrl-names = "default";
+	pinctrl-0 = <&bone_uart_2_pins>;
+	symlink = "bone/uart/2";
+};
+
+bone_uart_3: &ocp{
```

Comment:
Do you need anything more than a comment here?

Reply:
actually, we don't have this available on BBAI and If the node is not available overlay will cause a problem (ERROR: FDT not found!) so I created this. You remember we had that missing symbol problem, If I don't include this here that problem will occur.

Reply:
What is it that references this node to say FDT not found? Are you adding an overlay that tries to pick this up?

Reply:
I will certainly not use this as I know this is not available on BBAI. But, for someone new it's presence will save a lot of time as the only solution to recover your board is to flash a new image on your board if you accidentaly use uart_3 on BBAI.